contracts/src/interfaces/IChainlinkAggregatorV3.sol

```diff
@@ -2,6 +2,8 @@
 pragma solidity ^0.8.20;
 
 interface IChainlinkAggregatorV3 {
+    function aggregator() external view returns (address);
```

Comment:
natspec?

Reply:
None of the other interfaces have natspec. I don't think we need this here.